homeassistant/components/simplisafe/sensor.py

```diff
@@ -0,0 +1,80 @@
+"""Support for SimpliSafe freeze sensor."""
+from simplipy.entity import EntityTypes
+
+from homeassistant.const import DEVICE_CLASS_TEMPERATURE, TEMP_FAHRENHEIT
+from homeassistant.core import callback
+
+from . import SimpliSafeEntity
+from .binary_sensor import SimpliSafeSensorBattery
```

Comment:
We shouldn't depend on another platform in a platform. Make a common general base class if needed.